entry_points.txt

```diff
@@ -16,8 +16,7 @@ ansible_playbooks = cfme.ansible.playbooks:PlaybooksCollection
 ansible_repositories = cfme.ansible.repositories:RepositoryCollection
 ansible_tower_job_templates = cfme.ansible_tower.explorer:AnsibleTowerJobTemplatesCollection
 ansible_tower_jobs = cfme.ansible_tower.jobs:TowerJobsCollection
-ansible_tower_providers = cfme.ansible_tower.explorer:AnsibleTowerProvidersCollection
-ansible_tower_systems = cfme.ansible_tower.explorer:AnsibleTowerSystemsCollection
+ansible_tower_systems = cfme.infrastructure.config_management.config_systems.ansible_tower:AnsibleTowerSystemsCollection
```

Comment:
This collection name change has caused some PRT failures - there's a direct call to ansible_tower_providers via parametrization in cfme/tests/webui/test_advanced_search.py

Reply:
I wrote a GH issue and assigned it to @psimovec https://github.com/ManageIQ/integration_tests/issues/9723